it/uk/gov/hmrc/gatekeeperemail/repositories/FileUploadStatusRepositoryISpec.scala

```diff
@@ -74,7 +77,7 @@ class FileUploadStatusRepositorySpec
 
   "update a fileStatus to success" in {
     val fileReference = Reference(UUID.randomUUID().toString)
-    val fileStatus    = UploadInfo(fileReference, InProgress, LocalDateTime.now().truncatedTo(ChronoUnit.MILLIS))
+    val fileStatus    = UploadInfo(fileReference, InProgress, LocalDateTime.now(clock).truncatedTo(ChronoUnit.MILLIS))
```

Comment:
fixedclock has a method to replace all of this for LDT and for Instant and both truncate to millis.

Reply:
Hmmm I know what you mean but this repo has a FixedClock with none of that stuff
Where's the latest version of this util?